src/main/scala/edu/ie3/simona/model/participant/SystemParticipant.scala

```diff
@@ -173,7 +174,7 @@ abstract class SystemParticipant[
       nodalVoltage: Dimensionless
   ): Power => ReactivePower =
     qControl.activeToReactivePowerFunc(
-      sRated,
+      sRated.toPower,
```

Comment:
Note to self: Why don't we have to use power factor here?

Reply:
I adapted this method. Please have a look at the change.